app/models/early_career_teacher_profile.rb

```diff
@@ -1,6 +1,12 @@
 # frozen_string_literal: true
 
 class EarlyCareerTeacherProfile < ApplicationRecord
+  enum induction_programme_choice: {
+    core_induction_programme: "core_induction_programme",
+    full_induction_programme: "full_induction_programme",
+    not_yet_known: "not_yet_known",
+  }
```

Comment:
Are these all options we can get? :)

Probably not. Double check with what they are in R&P - we are, after all, returning the same things they have on their model from the api.

Reply:
The two others are 
`design_our_own: "design_our_own", 
no_early_career_teachers: "no_early_career_teachers",`

I thought we might not want to do anything with these... If you've no teachers, then you won't need mentors and if you're designing your own then you won't have access at anytime or need it. 

Happy to add them in if you think we should. 

I had it in my head yesterday that we wouldn't get these anyway but I can't remember why... maybe it was the programme types we initially had set up didn't include these.

Reply:
Well, whether we need them or not is a product decision. In current business logic we don't really use them.

But, from a technical perspective, it is possible that R&P will return any of those values on the user field, and crash our system when we try to save them. Because that's exactly the power and usefulness of an enum.

We should either add them to our end, or make sure that when we receive the data from R&P we cast them to things that our system understands.

How you approach it is up to you, all I want is for our app not to throw an error when it receives a valid R&P response :) You can use automated tests to simulate how the app will react to whatever you put on that field - just edit the fixture locally and see what happens when you run the tests for syncing users.